qcelemental/models/basis.py

```diff
@@ -37,6 +37,21 @@ def _check_coefficient_length(cls, v, values):
 
         return v
 
+    def nfunctions(self) -> int:
```

Comment:
I know you're against abbreviations, but `nbf` seems more descriptive

Reply:
On a single internal shell `nfunctions` seem more appropriate? I could go either way, just a shell isn't a full basis.

Reply:
good call. I'm back to `nfunctions` in this context